lib/src/utils/schema/widgetPropertyUpdaters.ts

```diff
@@ -119,11 +114,12 @@ function updateEnumTranslations(
   translationKey: string,
   locale: string,
 ) {
-  const translationState = getTranslationState(
-    verna.schema.translationSchema?.[locale],
-    values,
-    translationKey,
-  );
+  // Get the translation keys existing in every locale then filter the ones linked to this widget
+  const existingTranslationKeys = Object.entries(verna.schema.translationSchema || {})
+    .flatMap((locale) => Object.entries(locale[1]).map((a) => a))
+    .filter((translation) => translation[0].includes(translationKey));
```

Comment:
Hmm it's not to avoid empty translations it's just get the existing keys but that may be in every locale, we had a bug because we where searching only the current locale (so marked as new translation and override existing translations), i just take every translation keys in every locales, mixed them in one array, then filter to take only keys matching the widget to gain performance later.

I don't think it's a good idea to stop the user to put empty translations, i feel like every restriction we had in Verna will add complexity and not necessarily be a good thing for the user..

Maybe i didn't understood well your message, let me know